src/edit/UploadBtn.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <v-btn
+    @click="$refs.uploadInput.click()"
+    v-bind="$attrs"
+    color="primary darken-3"
+    text
+    class="text-uppercase">
+    <slot name="icon">
+      <v-icon color="secondary darken-1" class="mr-2">mdi-cloud-upload-outline</v-icon>
+    </slot>
+    {{ label }}
+    <input
+      ref="uploadInput"
+      @change="$emit('change', $event)"
+      type="file"
+      :accept="accept"
```

Comment:
Bound come props should come before static props.
```suggestion
      :accept="accept"
      type="file"
```